js/components/image.js

```
@@ -229,35 +229,34 @@ Fliplet.FormBuilder.field('image', {
-            $vm.isImageSizeExceeded = false;
-            $vm.value.push(file);
+          $vm.hasCorruptedImage = false;
+          $vm.isImageSizeExceeded = false;
 
-            if (addThumbnail) {
-              var scaledImage = loadImage.scale(img, options);
-              var imgBase64Url = scaledImage.toDataURL(mimeType, $vm.jpegQuality);
-              var flipletBase64Url = imgBase64Url + ';filename:' + file.name;
+          var scaledImage = loadImage.scale(img, options);
+          var imgBase64Url = scaledImage.toDataURL(mimeType, $vm.jpegQuality);
+          var flipletBase64Url = imgBase64Url + ';filename:' + file.name;
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)